### DISCUSSION TITLE
Support for PvP enemies as targets in skill calculations

### DISCUSSION THREAD

**Discussion by catgreasy** [lonqie/SchaleDB#39] [Ideas and Suggestions]
Would it be possible to add the ability to select students as enemies in the 'Skill Details' section? This would make damage calculation for 'Tactical Challenge' much easier. Thank you for considering this suggestion.

**Comment by lonqie** [comment]
Thanks for the suggestion.

If I made Striker students available as targets, I think for the best experience it would require the ability to set the star quantity, equipment and ideally student passives/buffs etc. to the target (to eliminate the need to set up using manual input buffs). T.S. vehicles and summons would also need to be considered.

It's a pretty significant change to the UI, as it was originally designed with primarily PvE content in mind.
